Menu/MenuBuilder.php

```diff
@@ -129,14 +78,4 @@ public function createSidebarMenu()
 
         return $event->getMenu();
     }
-
-    /**
-     * Sets the request the service.
-     *
-     * @param Request $request
-     */
-    public function setRequest(Request $request = null)
```

Comment:
Looks like it's not ok after all ? https://github.com/sonata-project/SonataAdminBundle/issues/3712